test/test_bot.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+import discord
+import discord.ext.test as dpytest
+
+
+@pytest.mark.asyncio(scope="session")
+async def test_enable_pickups(bot, helper):
+    await dpytest.message("!enable_pickups", member=0)
+
+    resp = await helper.message()
```

Comment:
why resp? it's a message, so call it that. :thonk:

`message.content` is nicer than `resp.content` too